README.rst

```diff
@@ -27,61 +27,48 @@ Take a look at the `Wiki`_ for more information.
 .. _`Wiki`: https://github.com/nens/ThreeDiToolbox/wiki
 
 
-Installing requirements
+Installation on windows
 -----------------------
 
-- QGIS 3 (64 bit)
-- pyqtgraph
-- sqlalchemy version 1.1.0 or higher
-- geoalchemy2 with custom modifications, source available here: https://github.com/nens/geoalchemy2
-- h5py (included for Windows)
+You need to install Qgis 3.4 (the "long term release"). Either the stand-alone
```

Comment:
3.4.5 is minimaal nodig toch?

"You need to install Qgis 3.4.5"
ipv
"You need to install Qgis 3.4"

Reply:
Done